__tests__/app.test.js

```diff
@@ -61,6 +61,183 @@ describe("/api/reviews", () => {
         expect(reviews).toBeSortedBy("created_at", { descending: true });
       });
   });
+
+  describe("/api/reviews (queries)", () => {
+    //Category
+    test("GET: 200 - RESPONDS WITH THE REVIEW ARRAY IF THE CATEGORY QUERY IS OMITTED", () => {
+      return request(app)
+        .get("/api/reviews?category")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toHaveLength(13);
+          reviews.forEach((review) => {
+            expect(review).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String), //SQL sends this back as a string
+            });
+          });
+          expect(reviews).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+  });
+
+  test("GET: 200 - RESPONDS WITH THE REVIEW ARRAY WITH THE SPECIFIED CATEGORY VALUE", () => {
+    return request(app)
+      .get("/api/reviews?category=euro game")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toHaveLength(1);
+        reviews.forEach((review) => {
+          expect(review).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: "euro game",
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(String), //SQL sends this back as a string
+          });
+        });
+      });
+  });
```

Comment:
Just missing one happy path - what happens if the category is valid but has no associated reviews? Looks like your code covers it, but best to make sure it's tested.